README.md

```diff
@@ -10,6 +10,15 @@ There are multiple ways to keep track of the on-going activity and participate:
 * multiple channels on the [Slack team for Readium](https://readium.slack.com) ([#readium-2](https://readium.slack.com/messages/readium-2/), [#r2-swift](https://readium.slack.com/messages/r2-swift/) and [#r2-java](https://readium.slack.com/messages/r2-java/))
 
 
+## Readium-2 architecture
+
+All Readium-2 implementations (mobile, desktop or Web app) can be divided in two parts, which may be running on different systems or written in completely different languages. 
+
+One contains the EPUB parser and the associated marshaling service, we call this backend part the "streamer". The other one contains the webview and associated rendering application, a frontend part that we call the "navigator". 
```

Comment:
It's not necessarily a webview. In the case of a Web app it could be just an iframe, and for audiobooks and comics it'll use something else.

There's also a typo: there's usually a - in "front-end" and "back-end".